src/Jab.FunctionalTests.MEDI/FactoryWithParametersTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
```

Comment:
Do you mind moving these tests to https://github.com/pakrym/jab/blob/main/src/Jab.FunctionalTests.Common/ContainerTests.cs 

I know it's huge but that file gets included in all test projects.